docs/faq.md

```diff
@@ -27,6 +27,8 @@ You can describe existing applications as Open Application Model applications. S
 
 Open Application Model is just an application model that, thanks to Rudr, runs on Kubernetes. You can bundle your Open Application Model applications as Helm charts and deploy them using Helm.
```

Comment:
Open Application Model is just an application model that, thanks to Rudr, runs on Kubernetes. You can bundle your Open Application Model applications as Helm charts and deploy them using Helm. Here is an example of [how to do that].